### ISSUE TITLE
README needs to have PyTorch version updated

### ISSUE THREAD

**Issue by convoliution** [deepsound-project/samplernn-pytorch#20]
The README still mentions PyTorch 0.1.12+ but since weight normalization was added to this code half a year ago this is no longer true as `torch.nn.utils.weight_norm` wasn't available until PyTorch 0.2
